telemetry/src/main/java/org/mozilla/telemetry/Telemetry.java

```diff
@@ -83,13 +84,27 @@ public Telemetry queueEvent(final TelemetryEvent event) {
         executor.submit(new Runnable() {
             @Override
             public void run() {
-                EventsMeasurement measurement = ((TelemetryEventPingBuilder) pingBuilders.get(TelemetryEventPingBuilder.TYPE))
-                        .getEventsMeasurement();
+                // We migrated from focus-event to mobile-event and unfortunately, this code was hard-coded to expect
+                // a focus-event ping builder. We work around this by checking our new hardcoded code first for the new
+                // ping type and then falling back on the legacy ping type.
```

Comment:
Just FYI, in telemetry-ios, we added a configuration item for apps to be able to specify which `PingBuilder` they want to use for recording events. Your approach works though too :-)